python/paddle/tensor/math.py

```diff
@@ -5727,3 +5727,61 @@ def i1e(x, name=None):
             type='i1e', inputs={'x': x}, outputs={'out': out}, attrs={}
         )
     return out
+
+
+def polygamma(x, n, name=None):
+    r"""
+    Calculates the polygamma of the given input tensor, element-wise.
+
+    Equation:
+        ..  math::
+
+            \Phi^n(x) = \frac{d^n}{dx^n} [\ln(\Gamma(x))]
```

Comment:
不用专门独立出一个 equation标题
![image](https://github.com/PaddlePaddle/Paddle/assets/70642955/f5918e03-9765-4366-b96a-c04c8e4b41eb)
```suggestion
    The equation is:
    
    .. math::
        \Phi^n(x) = \frac{d^n}{dx^n} [\ln(\Gamma(x))]
```

Reply:
好的，已修改